pkg/fleet/installer/default_packages.go

```diff
@@ -26,8 +26,8 @@ type Package struct {
 
 // PackagesList lists all known packages. Not all of them are installable
 var PackagesList = []Package{
-	{Name: "datadog-apm-inject", released: true, condition: apmInjectEnabled},
-	{Name: "datadog-apm-library-java", version: apmLanguageVersion, released: true, condition: apmLanguageEnabled},
+	{Name: "datadog-apm-inject", version: apmInjectVersion, released: true, condition: apmInjectEnabled},
+	{Name: "datadog-apm-library-java", version: apmJavaVersion, released: true, condition: apmLanguageEnabled},
```

Comment:
Do we want this in the install script as well?

Reply:
where is the code?